contentcuration/contentcuration/static/js/edit_channel/clipboard/hbtemplates/clipboard_list_item.handlebars

```diff
@@ -1,13 +1,20 @@
-<li id="item_{{index}}">
```

Comment:
Hrm - confusing. Which one are we using here - or are you using the ```<li>``` tags above to create an append point for this template when rendered?

Might be easier just to set the 'tagName' and the 'id' property of the view instead (see [here](http://backbonejs.org/#View-constructor) for more info):
`
'tagName': "li",
'id': function() {
    this.model.get("id");
}
`

Again, this is for the clipboard, so don't worry about this in the scope of this PR.